common/src/main/java/com/microsoft/identity/common/internal/cache/SharedPreferencesFileManager.java

```diff
@@ -113,13 +113,13 @@
             final String name,
             final IKeyAccessor encryptionManager) {
         if (encryptionManager == null) {
-            Logger.verbose(TAG, "Init: ");
+            Logger.verbose(TAG+"::constructor", "Init: ");
         } else {
-            Logger.verbose(TAG, "Init with storage helper:  " + TAG);
+            Logger.verbose(TAG+"::constructor", "Init with storage helper:  " + TAG);
```

Comment:
we dont need the last TAG

Reply:
Here: da1e62d1ddacd758eb5d0a6cedd9589521475ab8